modules/ossm-cert-manager-istio-csr-revisionbased-strategy.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="revision-based-istio-csr-installation_{context}"]
+= istio-csr installation: RevisionBased update strategy
+
+Follow this procedure if you are going to set `spec.updateStrategy` to `RevisionBased` when you create and install your `Istio` resource.
+
+.Procedure
+
+. Specify all the istio revisions to your `istio-csr` deployment.
```

Comment:
- Should "istio" be "Istio"?
- Is there anything else people should know in order to complete this step?